src/components/sharedDiary/SharedDiaryFunnel.tsx

```
@@ -0,0 +1,93 @@
+      //     defaultFile={file}
+      //     onNext={(nextContent, nextFile) =>
+      //       funnel.history.push("confirm", {
+      //         emotion,
+      //         content: nextContent,
+      //         file: nextFile,
+      //       })
+      //     }
+      //     onBack={() => funnel.history.pop()}
+      //   />
+      // );
+    }
```

Biome (2.1.2)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)